packages/@okta/vuepress-site/docs/reference/api/factor-admin/index.md

```diff
@@ -6,92 +6,89 @@ category: management
+| `symantec_vip`        | Symantec VIP         |
 
-### Factor Type
+### Factor type
 
 The following factor types are supported:
 
 | Factor Type           | Description                                                                                                         |
 | --------------------- | -----------                                                                                                         |
+| `question`            | Additional security question                                                                                        |
 | `sms`                 | SMS                                                                                                                 |
-| `token`               | A software or hardware one-time password [OTP](http://en.wikipedia.org/wiki/One-time_password) device               |
 | `token:software:totp` | Software [Time-based One-time Password (TOTP)](http://en.wikipedia.org/wiki/Time-based_One-time_Password_Algorithm) |
-| `question`            | Additional security question                                                                                        |
+| `token`               | A software or hardware one-time password [OTP](http://en.wikipedia.org/wiki/One-time_password) device               |
 
-### Provider Type
+### Provider type
 
 The following providers are supported:
 
 | Provider   | Description                   |
 | ---------- | ----------------------------- |
+| `GOOGLE`   | Google Integration            |
 | `OKTA`     | Okta                          |
 | `RSA`      | RSA SecurID Integration       |
 | `SYMANTEC` | Symantec VIP Integration      |
-| `GOOGLE`   | Google Integration            |
 
-### Links Object
+### Links object
 
-Specifies link relations (See [Web Linking](http://tools.ietf.org/html/rfc5988)) available for the current status of a factor using the [JSON Hypertext Application Language](http://tools.ietf.org/html/draft-kelly-json-hal-06) specification.  This object is used for dynamic discovery of related resources and lifecycle operations.
+Specifies link relations (see [Web Linking](http://tools.ietf.org/html/rfc5988)) available for the current status of a factor using the [JSON Hypertext Application Language](http://tools.ietf.org/html/draft-kelly-json-hal-06) specification. This object is used for dynamic discovery of related resources and lifecycle operations.
 
 | Link Relation Type | Description                        |
 | ------------------ | -----------                        |
-| self               | The actual factor                  |
 | activate           | Permits use of this factor for MFA |
 | deactivate         | Denies use of this factor for MFA  |
+| self               | The actual factor                  |
 
-> The Links Object is **read-only**
-
-## Factors Administration Operations
+> **Note:** The Links Object is **read-only**.
 
-### Get Org Factor
+## Factors Administration operations
```

Comment:
Should the "A" in "Administration" be lowercase, or is "Factors Administration" the proper noun?

Reply:
it's Factors Administration